src/item.h

```diff
@@ -307,7 +307,7 @@ class item : public visitable<item>
          * the extent of damage and burning (was created to sort by name without prefix
          * in additional inventory)
          */
-        std::string tname( unsigned int quantity = 1, bool with_prefix = true ) const;
+        std::string tname( unsigned int quantity = 1, bool with_prefix = false ) const;
```

Comment:
Why is the default value being changed from `true` to `false`?

Reply:
this was an old error i did.
there was an issue about it, and i later submitted a fix for this. it got corrected in mainline a while ago. since this PR started a while back and i did came back to it only lately, there was this old mistake remaining in there.